cmd/livepeer/livepeer.go

```diff
@@ -1018,6 +1018,10 @@ func getServiceURI(n *core.LivepeerNode, serviceAddr string) (*url.URL, error) {
 	}
 	addr := "https://" + strings.TrimSpace(string(body)) + ":" + RpcPort
 	inferredUri, err := url.ParseRequestURI(addr)
+	if err != nil {
+		glog.Error("Could not look up public IP address")
```

Comment:
Let's include the original error message here as well: 

`glog.Errorf("Could not look up public IP err=%v", err)`

Reply:
Sure, let me add it to the other error messages here as well.